cfgmgr/buffermgrdyn.cpp

```diff
@@ -265,6 +268,24 @@ string BufferMgrDynamic::getDynamicProfileName(const string &speed, const string
         buffer_profile_key = buffer_profile_key + "_" + gearbox_model;
     }
 
+    if (m_platform == "mellanox")
+    {
+        if ((speed != "400000") && (lane_count == 8))
```

Comment:
i am not sure i understand the logic behind this.
400g does not have a posix of 8 lanes but 100g may have but how come 100g is working with 8 lanes?

Reply:
A user can configure a 8 lane port to 100g.

Reply:
In nominal cases, 100 g port always have 4 lanes. In this case, we don’t add “8lane” to profile name.
But if user configured a 8 lanes port to 100g, we need to distinguish it from the normal case, so 8lane is added.